tests/validation/tests/v3_api/common.py

```diff
@@ -639,24 +686,59 @@ def validate_dns_record(pod, record, expected):
 
 
 def validate_dns_entry(pod, host, expected):
+    print("Expected target:", expected)
+
     # requires pod with `dig` available - TEST_IMAGE
     cmd = 'ping -c 1 -W 1 {0}'.format(host)
-    ping_output = kubectl_pod_exec(pod, cmd)
+    if is_windows_os:
+        cmd = 'powershell -NoLogo -NonInteractive -Command "& {{ ping -w 1 -n 1 {0} }}"'.format(host)
+
+    ping_result_pass = False
+    try_count = 1
+    if is_windows_os:
+        # Windows isn't good at ping, just increase the test times
```

Comment:
I got a little confused why Windows isn't good at ping. What would happen in Windows ping? I don't think we should to tolerate the ping lost in Windows.

Reply:
Rollbacked